R/rvar-summaries-over-draws.R

```diff
@@ -0,0 +1,187 @@
+# Summaries within array elements, over draws  --------------------------
+
+#' Summaries of random variables within array elements, over draws
+#'
+#' Compute summaries within elements of an [`rvar`] and over draws of each element,
+#' producing an array of the same shape as the input random variable (except in
+#' the case of `range()`, see 'Details').
+#'
+#' @param x an [`rvar`]
+#' @param ... further arguments passed to underlying functions (e.g., `base::mean()`
+#' or `base::median()`), such as `na.rm`.
+#'
+#' @details
+#'
+#' Summaries include expectations (`E()` or `mean()`), probabilities (`Pr()`),
+#' medians (`median()`), spread (`variance()`, `sd()`, `mad()`), sums and
```

Comment:
with the latest changes, can we add `var` to the list as well?

Reply:
good point!